doc/api-extensions.md

```diff
@@ -2515,3 +2515,8 @@ contains the total available logical CPUs available when LXD started.
 ## `vm_limits_cpu_pin_strategy`
 
 Adds a new {config:option}`instance-resource-limits:limits.cpu.pin_strategy` configuration option for virtual machines. This option controls the CPU pinning strategy. When set to `none`, CPU auto pinning is disabled. When set to `auto`, CPU auto pinning is enabled.
+
+## `gpu_cdi_mode`
```

Comment:
"Mode" seems redundent here. How about just gpu_cdi or gpu_cdi_container¿

Reply:
OK for `gpu_cdi`. fixed